runelite-client/src/main/java/net/runelite/client/plugins/hiscore/NameAutocompleter.java

```diff
@@ -145,7 +158,7 @@ public void keyTyped(KeyEvent e)
 
 	private void newAutocomplete(KeyEvent e)
 	{
-		final JTextComponent input = (JTextComponent)e.getSource();
+		final JTextComponent input = (JTextComponent) e.getSource();
```

Comment:
There are a lot of irrelevant changes in this, are you using our style format file?

Reply:
Yes, I am using the style found in the wiki. I'll change them back